package.json

```diff
@@ -3006,6 +3006,18 @@
 						"markdownDescription": "Specifies the number of items to show in a each page when paginating a graph list. Use 0 to specify no limit",
 						"scope": "window",
 						"order": 20
+					},
+					"gitlens.graph.columnColors": {
+						"type": "array",
+						"items": {
+							"type": "string"
+						},
+						"default": ["#15a0bf", "#0669f7", "#8e00c2", "#c517b6", "#d90171", "#cd0101", "#f25d2e", "#f2ca33", "#7bd938", "#2ece9d"],
```

Comment:
🤔 What would happen if the user enters an incorrect color here?  

I think the component doesn't check if the color is valid or not. We need to take a look at this and if it is the case, add code to protect this on the library side (on a new JIRA task).

Reply:
the graph breaks, I'll investigate forcing formatting the columns as colors or checking the colors before I add them to the CSS variables as well as creating a new issue to validate the css variables.

Reply:
https://gitkraken.atlassian.net/browse/GITLENS-283

Reply:
@ericf-axosoft I added a regex on this side to ensure the column color array has an rgb string. We should still do the library-side fix as well though